resources/geocoding/en/98.txt

```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2012 The Libphonenumber Authors
+# Copyright (C) 2016 The Libphonenumber Authors
```

Comment:
So you can actually leave this as 2012 since the file was originally created then. We would only had the current calendar year for a new file.